infra/modules/gcp-bulk/main.tf

```diff
@@ -18,7 +18,7 @@ module "psoxy-package" {
   source = "../psoxy-package"
 
   implementation     = "gcp"
-  path_to_psoxy_java = "../../../java"
+  path_to_psoxy_java = "${var.psoxy_basedir}/java"
```

Comment:
again, move up to config. needed only once for `gcp`

Reply:
I'll leave the *-bulk cases for another refactor.